### DISCUSSION TITLE
Unable to find uploaded build at App Store Connect

### DISCUSSION THREAD

**Discussion by xs2bas** [airsdk/Adobe-Runtime-Support#601] [Q&A]
For the time being I still use version 32.0.0.144 of the SDK for uploading updates of app to Apple. 
Yesterday I uploaded a new update, but it didn't appear in the 'Build'-section at App Store Connect, so than I tried building an app using SDK 33.1.1.98 or 33.1.1.86, but I receive this message when building with Animate:

---------------------------
Adobe Animate
---------------------------
Error creating files.

Warning: Using default implementation for native extension 'com.distriqt.square.picasso'
Warning: Resource de.lproj has been skipped because of mismatch with supported languages information in application descriptor.
Warning: Resource en.lproj has been skipped because of mismatch with supported languages information in application descriptor.
Warning: Resource es.lproj has been skipped because of mismatch with supported languages information in application descriptor.
Warning: Resource fr.lproj has been skipped because of mismatch with supported languages information in application descriptor.
ld: library not found for -lclang_rt.ios
ld: library not found for -lclang_rt.ios
Compilation failed while executing : ld64

---------------------------
OK   
---------------------------

... is there a special way to configure this SDK? The header of the application-descriptor is set like this:
<application xmlns="http://ns.adobe.com/air/application/33.1">

... and I updated all the Distriqt-ANEs I use

**Comment by marchbold** [comment]
See https://github.com/Gamua/Adobe-Runtime-Support/issues/113

You need to add `libclang_rt.ios.a` into your AIR SDK

**Comment by xs2bas** [comment]
Thanks, that solved the problem! ... well, partly, because I uploaded the new build to Apple, but it still does not show up at 'Build' at App Store Connect when I want to deploy that new version ... 
It finished processing, and I only received the warning that I always receive: 
- ITMS-90078: Missing Push Notification Entitlement 
- ITMS-90191: Missing beta entitlement 

They are only warnings, so I guess that;s not the problem ...

**Comment by xs2bas** [comment]
Did anybody else experience this problem? I tried for two different apps, using the latest SDK ... after i successfully uploaded the app it doesn't appear in the list of builds to select for publishing. Trying for days now, but don't know how to solve it :(
I found some topics online with people having the same problem, which has maybe something to do with a UIWebView reference ...
